source/gloperate/source/stages/base/ShaderLoaderStage.cpp

```diff
@@ -0,0 +1,50 @@
+
+#include <gloperate/stages/base/ShaderLoaderStage.h>
+
+#include <glbinding/gl/gl.h>
+
+#include <globjects/Shader.h>
+
+#include <gloperate/gloperate.h>
+
+
+namespace gloperate
+{
+
+
+CPPEXPOSE_COMPONENT(ShaderLoaderStage, gloperate::Stage)
+
+
+ShaderLoaderStage::ShaderLoaderStage(Environment * environment, const std::string & name)
+: Stage(environment, name)
+, filePath("filePath", this)
+, source("source", this)
+, type("type", this)
+, m_file(nullptr)
+, m_extensionToType({
+    {"vert", gl::GL_VERTEX_SHADER},
+    {"tec", gl::GL_TESS_CONTROL_SHADER},
+    {"tes", gl::GL_TESS_EVALUATION_SHADER},
```

Comment:
Are these file extensions correct? Please review. Also this does not have to be a 1:1 relation. Are there other extensions which should be matched with existing types?